deployment/ansible/group_vars/all

```diff
@@ -0,0 +1,7 @@
+java_version: "7u131*"
+terraform_version: "0.9.3"
+awscli_version: "1.11.*"
+aws_profile: "geotrellis-site"
+docker_version: "1.10.*"
+docker_compose_version: "1.9.*"
+ansible_user: "vagrant"
```

Comment:
This is better named `docker_user`. Since Ansible is configured to use `sudo` during provisioning, the `vagrant` user doesn't get added to the `docker` group unless we pass the username in as a variable.